depends/packages/proton.mk

```diff
@@ -11,7 +11,7 @@ define $(package)_preprocess_cmds
 endef
 
 define $(package)_config_cmds
-  cd build; cmake .. -DCMAKE_CXX_STANDARD=11 -DCMAKE_INSTALL_PREFIX=/ -DSYSINSTALL_BINDINGS=ON -DCMAKE_POSITION_INDEPENDENT_CODE=ON -DBUILD_PYTHON=OFF -DBUILD_PHP=OFF -DBUILD_JAVA=OFF -DBUILD_PERL=OFF -DBUILD_RUBY=OFF -DBUILD_JAVASCRIPT=OFF -DBUILD_GO=OFF -DBUILD_STATIC_LIBS=ON
+  cd build; cmake .. -DCMAKE_CXX_STANDARD=11 -DCMAKE_INSTALL_PREFIX=/ -DSYSINSTALL_BINDINGS=ON -DCMAKE_POSITION_INDEPENDENT_CODE=ON -DBUILD_PYTHON=OFF -DBUILD_RUBY=OFF -DBUILD_GO=OFF -DBUILD_STATIC_LIBS=ON -DLIB_SUFFIX= -DENABLE_JSONCPP=
```

Comment:
Why are these changes necessary here? I am not against updating proton version, but why are compile flags changing? Did this come from KMD or BTC or ZEC?

Reply:
I've replaced complete file, haven't edited line by line. Mostly it's order of flags that's changed, not the flags themselves. Taken from zcash depends.

Reply:
`-DBUILD_JAVA=OFF -DBUILD_PERL=OFF -DBUILD_PHP=OFF` was lost, but we are deleting Proton, so it doesn't matter 🎉

Reply:
proton removed